assets/airports/airportLoadList.js

```diff
@@ -72,6 +72,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             wip: true
         },
         {
+            icao: 'efhk',
+            level: 'easy',
+            name: 'Helsinki Airport',
```

Comment:
Up to you how you want it, but wouldn't Helsinki-Vantaa Airport look a bit more unique?